crates/ruff_linter/src/rules/pycodestyle/snapshots/ruff_linter__rules__pycodestyle__tests__E305_E30.py.snap

```diff
@@ -11,14 +11,13 @@ E30.py:806:1: E305 [*] Expected 2 blank lines after class or function definition
     = help: Add missing blank line(s)
 
 ℹ Safe fix
-803 803 |     # comment
-804 804 | 
-805 805 |     # another comment
-    806 |+
-    807 |+
-806 808 | fn()
-807 809 | # end
-808 810 | 
+800 800 | def fn():
+801 801 |     print()
+802 802 | 
+    803 |+
+803 804 |     # comment
+804 805 | 
+805 806 |     # another comment
```

Comment:
I think this is incorrect. The blank lines got added between the last statement and the comment within the same block (function body).